### DISCUSSION TITLE
URLs with special characters cannot be added

### DISCUSSION THREAD

**Discussion by nasirhemed** [plausible/analytics#1065] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
**Describe the bug**
Some URLs have special characters in them such as tilde, spaces, quotes, ... etc.  

For example: https://www.cs.toronto.edu/~hinton/

If I try to add such URLs, I get an error saying: "only letters, numbers, slashes and period allowed"

Would it be possible to always encode the user's url input. That way, we don't have to inform the user that the url is invalid.

**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.

**Comment by metmarkosaric** [comment]
**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.
